CMakeLists.txt

```diff
@@ -73,8 +73,7 @@ add_project_dependency(state-observation REQUIRED)
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/CMakeModules)
 
-# hpp-spline (defines target: mc_rtc_3rd_party::hpp-spline)
-find_package(mc_rtc_3rd_party_hpp-spline REQUIRED)
+find_package(mc_rtc_3rd_party_ndcurves REQUIRED)
```

Comment:
If the extra module is not needed we can simply use `add_project_dependency(ndcurves REQUIRED)`

Reply:
`ndcurvesConfig/ndcurvesTargets.cmake` does indeed exist.
However it fails because it  tries to look for `pinocchio`, which ought to be optional, but does so using `find_dependency(pinocchio QUIET)`. We could either get it fixed on `ndcurves` side, or keep the current extra module.

Reply:
Oh, indeed!

I've opened https://github.com/loco-3d/ndcurves/pull/67 hopefully it resolves this issue